### PR TITLE
Add comment on environment version difference

### PR DESCRIPTION
**Pull request by marioyc**

### CODE REVIEW

File: examples/mujoco/reproduction/ppo/README.md [hunk 1]
```diff
@@ -28,6 +28,7 @@ To view the full list of options, either view the code or run the example with t
 ## Known differences
 
 - While the original paper initialized weights by normal distribution (https://github.com/Breakend/baselines/blob/50ffe01d254221db75cdb5c2ba0ab51a6da06b0a/baselines/ppo1/mlp_policy.py#L28), we use orthogonal initialization as the latest openai/baselines does (https://github.com/openai/baselines/blob/9b68103b737ac46bc201dfb3121cfa5df2127e53/baselines/a2c/utils.py#L61).
+- We used version v2 of the environments whereas the original results where reported for version v1, however this doesnt' seem to introduce significant differences: https://github.com/openai/gym/pull/834
```
Comment (muupan) [inline examples/mujoco/reproduction/ppo/README.md]: s/where/were/ ?

File: examples/mujoco/reproduction/trpo/README.md [hunk 2]
```diff
@@ -25,19 +25,23 @@ python train_trpo.py [options]
 
 To view the full list of options, either view the code or run the example with the `--help` option.
 
+## Known differences
+
+- We used version v2 of the environments whereas the original results where reported for version v1, however this doesnt' seem to introduce significant differences: https://github.com/openai/gym/pull/834
```
Comment (muupan) [inline examples/mujoco/reproduction/trpo/README.md]: Ditto

File: examples/mujoco/reproduction/ppo/README.md (hunk 1 above)
Comment (marioyc) [reply examples/mujoco/reproduction/ppo/README.md]: Fixed

File: examples/mujoco/reproduction/trpo/README.md (hunk 2 above)
Comment (marioyc) [reply examples/mujoco/reproduction/trpo/README.md]: Fixed

File: examples/mujoco/reproduction/ppo/README.md [hunk 3]
```diff
@@ -28,6 +28,7 @@ To view the full list of options, either view the code or run the example with t
 ## Known differences
 
 - While the original paper initialized weights by normal distribution (https://github.com/Breakend/baselines/blob/50ffe01d254221db75cdb5c2ba0ab51a6da06b0a/baselines/ppo1/mlp_policy.py#L28), we use orthogonal initialization as the latest openai/baselines does (https://github.com/openai/baselines/blob/9b68103b737ac46bc201dfb3121cfa5df2127e53/baselines/a2c/utils.py#L61).
+- We used version v2 of the environments whereas the original results were reported for version v1, however this doesnt' seem to introduce significant differences: https://github.com/openai/gym/pull/834
```
Comment (muupan) [inline examples/mujoco/reproduction/ppo/README.md]: s/doesnt'/doesn't/ ?

Comment (marioyc) [reply examples/mujoco/reproduction/ppo/README.md]: Fixed

**Comment by marioyc** [comment]
Looks like test_stateless_recurrent_sequential is flaky

**Pull request merged**